dd-bridge-android/src/main/kotlin/com/datadog/android/bridge/internal/BridgeSdk.kt

```diff
@@ -57,6 +57,11 @@ internal class BridgeSdk(
             crashReportsEnabled = configuration.nativeCrashReportEnabled ?: false,
             rumEnabled = true
         )
+            .setAdditionalConfiguration(
+                configuration.additionalConfig
+                    ?.filterValues { it != null }
```

Comment:
this is needed because on the receiver (`dd-sdk-android`) side the type of `additionalConfiguration` is `Map<String, Any>?` and not `Map<String, Any?>?`. Anyway, having nullable values in the `Map` is not useful in my opinion and doesn't make sense.